src/components/Badge/Badge.scss

```diff
@@ -19,12 +18,14 @@ $pip-spacing: ($height - $pip-size) / 2;
   @include pip-color(var(--p-icon, color('ink', 'lightest')));
   display: inline-flex;
   align-items: center;
+  min-width: $height;
+  min-height: $height;
```

Comment:
Here we set the desired height of the badge, without preventing it from growing if the text wraps